scripts/test.sh

```diff
@@ -0,0 +1 @@
+jest --env=jsdom ${1:-src}
```

Comment:
No need for this file (see my previous comment).

Reply:
Sounds good